src/test/mmap_replace_most_mappings.c

```diff
@@ -5,7 +5,11 @@
 #define MAX_ERRNO 4095
 
 #define RR_PAGE_ADDR 0x70000000
+#ifdef __aarch64__
+#define RR_THREAD_LOCALS_PAGE_ADDR 0x70010000
+#else
 #define RR_THREAD_LOCALS_PAGE_ADDR 0x70001000
+#endif
```

Comment:
Likewise this could be `RR_PAGE_ADDR + page_size()`?

Reply:
This is in the test so it's somewhat different. It would have to be `rr_page_size()` as well.